pallene/coder.lua

```diff
@@ -196,10 +196,11 @@ end
 local function check_no_metatable(src, loc)
     return (util.render([[
         if ($src->metatable) {
-            pallene_runtime_array_metatable_error(L, $line);
+            pallene_runtime_array_metatable_error(L, $file, $line);
         }
     ]], {
         src = src,
+        file = C.string(loc.file_name),
```

Comment:
It is a bit distracting to see these string literals repeated at every syntax error. Would it look better if we created a  `#define` at the top of the file to hold the file name?